pages/docs/reference/js-project-setup.md

```diff
@@ -365,7 +365,21 @@ The Kotlin/JS Gradle plugin automatically generates a standard webpack configura
 at build time which you can find the at `build/js/packages/projectName/webpack.config.js`.
 
 The most common webpack adjustments can be made directly via the
-`kotlin.js.browser.webpackTask` configuration block in the Gradle build file.
+`kotlin.js.browser.webpackTask` configuration block in the Gradle build file. These are:
+- `outputFileName` - the name of the webpacked output file. It will be generated in `<projectDir>/build/distibution/` after
+an execution of a webpack task. The default value is the project name.
+- `output.libraryTarget` - the module system for the webpacked output. Learn more about available module systems for
+Kotlin/JS projects [here](js-modules.html). The default value is `umd`.
```

Comment:
Please avoid using here as linked text. It doesn't add any value. Make real text `available module systems for Kotlin/JS` a link.